src/AstTypeAnalysis.h

```diff
@@ -90,7 +93,7 @@ class TypeAnalysis : public AstAnalysis {
      * @return a map mapping each contained argument to a a set of types
      */
     static std::map<const AstArgument*, TypeSet> analyseTypes(const TypeEnvironment& env,
-            const AstClause& clause, const AstProgram* program, bool verbose = false);
+            const AstClause& clause, const AstProgram* program, std::ostream* logs = NULL);
```

Comment:
nullptr

Reply:
Fixed in e16185d10dee7789f65b1c444dce58994ab769ea